health/grpc_health_v1/health_grpc.pb.go

```diff
@@ -43,6 +43,10 @@ const (
 // HealthClient is the client API for Health service.
 //
 // For semantics around ctx use and closing/ending streaming RPCs, please refer to https://pkg.go.dev/google.golang.org/grpc/?tab=doc#ClientConn.NewStream.
+
```

Comment:
This should have a `//` on it so it's part of the other comments.  (Please fix the codegen and regenerate.)